clang/lib/Basic/Targets/PPC.h

```diff
@@ -80,6 +80,7 @@ class LLVM_LIBRARY_VISIBILITY PPCTargetInfo : public TargetInfo {
   bool IsISA3_0 = false;
   bool IsISA3_1 = false;
   bool HasQuadwordAtomics = false;
+  bool HasAIXShLibTLSModelHeuristic = false;
```

Comment:
Hi Kai, Thank you for pointing out this.

I checked the code and it seems this flag is being used by backend in various places (for example: `PPCTargetLowering::LowerGlobalTLSAddressAIX`). You referred to another example that this flag is redundant, may I know the details and I will cross check with that scenario?

Reply:
The variable defined here is only visible to clang and clang doesn't use this flag further. What backend sees is the feature string and set corresponding variables in backend accordingly.

Reply:
Code in `PPCTargetLowering::LowerGlobalTLSAddressAIX` like
```
Subtarget.hasAIXSmallLocalExecTLS()
```
which is generated in `PPCGenSubtargetInfo.inc`, whose value is initialized by feature string `aix-small-local-exec-tls` in LLVM IR rather than the variable defined in clang.